src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/EquatableAnalyzerTests.cs

```diff
@@ -370,6 +371,21 @@ public class D : C<D>
             VerifyCSharp(code);
         }
 
+        [Fact, WorkItem(2324, "https://github.com/dotnet/roslyn-analyzers/issues/2324")]
+        public void CA1066_CSharp_RefStruct_NoDiagnostic()
+        {
+            VerifyCSharp(@"
+public ref struct S
+{
+    public override bool Equals(object other)
+    {
+        return false;
+    }
+}
+",
+            parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
```

Comment:
I don't know what's best between `CSharp8`, `Latest` or some other option that would automatically take care of testing with each and every new version starting at 8.

Reply:
`CSharp8` is good. `Latest` should never be used anywhere (it's not a version).